Full Traceback (most recent call last):
  File "/opt/miniconda3/envs/snakemake/lib/python3.9/site-packages/snakemake/__init__.py", line 591, in snakemake
    workflow.include(
  File "/opt/miniconda3/envs/snakemake/lib/python3.9/site-packages/snakemake/workflow.py", line 1173, in include
    exec(compile(code, snakefile, "exec"), self.globals)
  File "/home/jeszyman/repos/cfdna-cna/workflow/int_test.smk", line 33, in <module>
  File "/opt/miniconda3/envs/snakemake/lib/python3.9/site-packages/snakemake/workflow.py", line 1173, in include
    exec(compile(code, snakefile, "exec"), self.globals)
  File "/home/jeszyman/repos/cfdna-cna/workflow/cfdna_wgs_cna.smk", line 86, in <module>
KeyError: 'wig_dir'

KeyError in line 42 of /home/jeszyman/repos/cfdna-cna/workflow/cfdna_wgs_cna.smk:
'wig_dir'
  File "/home/jeszyman/repos/cfdna-cna/workflow/int_test.smk", line 27, in <module>
  File "/home/jeszyman/repos/cfdna-cna/workflow/cfdna_wgs_cna.smk", line 42, in <module>
